### ISSUE TITLE
epub status screen should display page number

### ISSUE THREAD

**Issue by dracodoc** [koreader/kindlepdfviewer#422]
When reading epub with kpv, the status screen only display the percentage of current reading location. It's understandable because epub don't have a fixed "page" concept, the page number may change with different font size.

However, the bookmarks in epub only showed some page number. Thus when user check the bookmark list, it's impossible to relate current reading location with bookmark pages.

I suggest either add page number in status screen, or add percentage to bookmark list to make these two location system match.

**Comment by houqp** [comment]
Good point, I would suggest using percentage in bookmark list and all other lists.

**Comment by dracodoc** [comment]
Another related problem is, the alt+K/L function of jumping between bookmarks doesn't work in page mode. I have to change to scroll mode to use the feature. I'm not sure if this limit is built-in of page mode or it's is a mode.
